2269/2/rapers_list.rb

```diff
@@ -0,0 +1,22 @@
+# :reek:DuplicateMethodCall
+# :reek:UtilityFunction
+# :reek:TooManyStatements
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
+def get_rapers(destination)
+  hash = {}
+  Dir.entries(destination).each do |file|
+    next if ['.', '..'].include?(file)
```

Comment:
you can use Dir method that does not include directories.

Reply:
this issue fixed